### DISCUSSION TITLE
Eager to Help

### DISCUSSION THREAD

**Discussion by suzy-g38** [pbclife/gitopener.vercel.app#150] [General]
Hey, guys @priyankarpal @sboy99  if you need any extra hands I am always there and will be glad if I can help you, people.

**Comment by sboy99** [comment]
Can you suggest improvements??

**Reply by suzy-g38** [reply]
One suggestion I want to say, I was thinking of raising an issue for that, but anyway.....when we are selecting the dark theme or light theme button it may be hard sometime for the user to understand if they have chosen the dark theme button or light theme button.....So if you guys can give some special effects or color changes on the selected button that can give some clarity.
![image](https://user-images.githubusercontent.com/62666287/232311243-26ac49b0-48a5-48bd-9f33-54bc4a8cbfc0.png)
Here we can clearly understand it is dark theme mode.
![image](https://user-images.githubusercontent.com/62666287/232311396-52d22d48-2fa8-4fdf-9483-a89d71310e8f.png)
but from here if we just see the button you don't know which one you have selected.
So, if you can give some slider type that can really help. or if you can choose another way to do that......more or less this is my suggestion.

**Reply by priyankarpal** [reply]
what if we add gradient animated color?

**Reply by suzy-g38** [reply]
yess that can work too...just if we make them any different that will suffice

**Reply by priyankarpal** [reply]
> yess that can work too...just if we make them any different that will suffice

ok also share a demo

**Comment by priyankarpal** [comment]
In that project we're using SVG icons that's why the project's code getting too long. So we can create custom components to add icons.

example
```
const GitHubIcon = () => {
    return (
        <>
           SVG CODE
        </>
    )
}
```

 User can add icons using just `<GitHubIcon/>`

**Reply by suzy-g38** [reply]
okay Can I do this one?

**Reply by priyankarpal** [reply]
> okay Can I do this one?

yeah sure

**Reply by suzy-g38** [reply]
Hey, @priyankarpal @sboy99 I have tried to add icons in a different way, attaching the code-sandbox link below please review it.
https://codesandbox.io/s/ecstatic-montalcini-n1vxn7?file=/src/App.js
